apps/emqx_bridge_azure_event_hub/src/emqx_bridge_azure_event_hub_action_info.erl

```diff
@@ -20,3 +22,19 @@ action_type_name() -> azure_event_hub_producer.
 connector_type_name() -> azure_event_hub_producer.
 
 schema_module() -> emqx_bridge_azure_event_hub.
+
+action_to_bridge_v1_fixup(Config) ->
+    rename(<<"parameters">>, <<"kafka">>, Config).
+
+rename(OldKey, NewKey, Map) ->
+    case maps:find(OldKey, Map) of
+        {ok, Value} ->
+            maps:remove(OldKey, maps:put(NewKey, Value, Map));
```

Comment:
Nit:

```suggestion
            maps:put(NewKey, Value, maps:remove(OldKey, Map));
```

Because renaming a key to itself would remove it from the result otherwise.

Maybe this could be added to `emqx_utils_maps`.